google/auth/_cloud_sdk.py

```diff
@@ -105,18 +105,14 @@ def get_project_id():
     try:
         # Ignore the stderr coming from gcloud, so it won't be mixed into the output.
         # https://github.com/googleapis/google-auth-library-python/issues/673
-        output = _run_subprocess_ignore_stderr((command,) + _CLOUD_SDK_CONFIG_COMMAND)
-    except (subprocess.CalledProcessError, OSError, IOError):
-        return None
-
-    try:
-        configuration = json.loads(output.decode("utf-8"))
-    except ValueError:
-        return None
+        project = _run_subprocess_ignore_stderr(
+            (command,) + _CLOUD_SDK_CONFIG_GET_PROJECT_COMMAND
+        )
 
-    try:
-        return configuration["configuration"]["properties"]["core"]["project"]
-    except KeyError:
+        # Turn bytes into a string and remove "\n"
+        project = _helpers.from_bytes(project).strip()
```

Comment:
Do we need to do any parsing? This command outputs this for me:

```
>>> subprocess.run(['gcloud', 'config', 'get', 'project'])
Your active configuration is: [test]
stellar-day-254222
```

Seems that the first line may cause some issues, or is it okay?

Reply:
The first line won't show up because it writes to stderr

```
import subprocess
from google.auth import _cloud_sdk

output = subprocess.check_output(('gcloud', 'config', 'get', 'project'))
print(output)
print('---------------')
print(_cloud_sdk.get_project_id())
```

result is:
```
Your active configuration is: [foo]
b'sijun-auth\n'
---------------
sijun-auth
```

Reply:
Missed that! Thanks!